src/main/java/seedu/address/model/person/Phone.java

```diff
@@ -4,7 +4,7 @@
 import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's phone number in the address book.
+ * Represents a person's phone number in the address book.
```

Comment:
I think this comment should remain as 'Represent a Person's...' instead of 'person'. The reason is that 'Person' represents the class, while 'person' is an English term. This comment is meant to be a technical description of what this class is. In this case, this class represents the phone number of Person objects. Therefore, I think 'Person' is a more apt choice.